src/main/java/ch/puzzle/eft/service/ExamService.java

```diff
@@ -64,6 +65,7 @@ public List<ExamModel> getMatchingExams(String examNumber, String matriculationN
         }
         return matchingFiles.stream()
                             .map(ExamModel::new)
+                            .sorted(Comparator.comparing(ExamModel::getSubjectName, String.CASE_INSENSITIVE_ORDER))
```

Comment:
Case INsensitive ist OK. Die Umlaute werden aber immer noch am Schluss der Liste angefügt, was im deutschsprachingen Raum ungewöhnlich ist. 

Die Umlaute müssen ebenfalls in der richtigen Reihenfolge erscheinen, d.h. ä bei a, ö bei o und ü bei u.